style/style.css

```diff
@@ -1,3 +1,86 @@
-body {
-    margin: 0;
-}
\ No newline at end of file
+        @import url(https://fonts.googleapis.com/css?family=Fira+Sans:100,200,300italic,regular,600,600italic,);
+        body {
+            margin: 0;
+            box-sizing: border-box;
+            display: flex;
+        }
+        
+        .triangle_conteiner {
+            position: relative;
+            overflow: hidden;
+        }
+        
+        .cover {
+            position: absolute;
+            top: 136px;
+            right: -146px;
+            width: 450px;
+            height: 300px;
+            background-color: #fff;
+            transform: rotate(-27deg);
+        }
+        
+        .wrapper {
+            margin: 50px;
+            width: 290px;
+            height: 570px;
+            border-radius: 25px;
+            background: #FFFFFF;
+            border: 1px solid #BDBDBD;
+            overflow: hidden;
+        }
+        
+        .made {
+            font-family: 'Fira Sans';
+            font-style: normal;
+            font-weight: 300;
+            font-size: 14px;
+            line-height: 18px;
+            color: #333333;
+        }
+        
+        .card_container {
+            padding: 0 23.5px 24px;
+            text-align: center;
+            font-family: 'Fira Sans';
+            font-style: normal;
+            font-weight: 600;
+            font-size: 24px;
+            line-height: 32px;
+        }
+        
+        .regular {
+            box-sizing: border-box;
+            display: flex;
+            flex-direction: row;
+            color: #333333;
+            padding: 9px 16px;
+            gap: 27px;
+            width: 243px;
+            height: 40px;
+            border: 1px solid #4F4F4F;
+            border-radius: 25px;
+            margin: 0 0 10px;
+            cursor: pointer;
+        }
+        
+        .Cart {
+            display: flex;
+            flex-direction: row;
+            justify-content: center;
+            align-items: center;
+            color: #FFFFFF;
+            padding: 10px 24px 11px;
+            gap: 10px;
+            width: 243px;
+            height: 40px;
+            background: #F44A25;
+            border-radius: 25px;
+            font-family: 'Fira Sans';
+            font-style: normal;
+            cursor: pointer;
+        }
```

Comment:
The same mistakes here about 
flex-diraction
gap
and you dont separate your properties